packages/jsapi-utils/src/MessageUtils.ts

```diff
@@ -25,34 +21,14 @@ export const BROADCAST_LOGIN_MESSAGE = 'io.deephaven.broadcast.Login';
  */
 export const BROADCAST_LOGOUT_MESSAGE = 'io.deephaven.broadcast.Logout';
 
-export interface Message<T = unknown> {
-  id: string;
-  message: string;
-  payload?: T;
-}
-
-export interface BroadcastLoginMessage extends Message<void> {
+export interface BroadcastLoginMessage extends PostMessage<void> {
   message: typeof BROADCAST_LOGIN_MESSAGE;
 }
 
-export interface BroadcastLogoutMessage extends Message<void> {
+export interface BroadcastLogoutMessage extends PostMessage<void> {
   message: typeof BROADCAST_LOGOUT_MESSAGE;
 }
 
-export interface Response<T = unknown> {
-  id: string;
-  payload: T;
-}
-
-export function isMessage(obj: unknown): obj is Message {
-  const message = obj as Message;
-  return (
-    message != null &&
-    typeof message.id === 'string' &&
-    typeof message.message === 'string'
-  );
-}
-
```

Comment:
Removing all this stuff is a breaking change. Deprecate it instead (and re-export from utils where possible).